tests/integration/api/xai/test_api_xai_sanity.py

```diff
@@ -38,27 +38,37 @@
 
 save_model_to = "/tmp/otx_xai/"
```

Comment:
Is there any specific reason in preferring a hardcoded path instead of tempfile.TemporaryFile?

Reply:
No there is no, fixed!